src/main/java/bio/terra/service/filedata/flight/ingest/IngestCopyLoadHistoryToBQStep.java

```diff
@@ -66,6 +67,7 @@ public StepResult doStep(FlightContext context) {
                         loadTag,
                         loadTime,
                         loadHistoryArray);
+                    TimeUnit.SECONDS.sleep(5);
```

Comment:
Maybe add a comment about why 5 seconds?

In our target 150K load with 1000 chunk size, we would be sleeping for ~12 minutes of elapsed time. So if this can be lower, it would make a real difference to the elapsed load time.

Reply:
Not totally sure I approached this correctly, so open to more suggestions:
1. Time benchmark:  I searched the BQ logs in the dev-data project and the largest query I found took about 3 seconds. Smaller inserts into the staging table take less than 1 second, and the majority I found took under 2 seconds. But, this is based on a very limited number of logs. 
2. Selecting sleep time: I was thinking if we matched the longest time a query should take (3 seconds), then we should be pretty safe in avoiding rate limit errors. But, I'm thinking we could probably lower to 1 or 2 seconds and still avoid the error. 

So, since this selection is impactful to performance (sleep time selection adds up and can have minutes-long impact on ingest time) and query performance varies, I went ahead and moved this into a config variable.

Reply:
Reading the [Quotas and Limits](https://cloud.google.com/bigquery/quotas) section, it is hard to understand which limit we are hitting and how we could possibly be hitting it. No per-table quota seems low enough. _Maybe_ we are hitting the maximum insert requests per second per project of 10. That seems hard to achieve given how we run our tests and segment the projects.

Perhaps we should just accept that this quota error will happen sometimes and write compensating code. For example, wrapping this in some kind of retry. If we hit the quota error, we do some random backoff with a mean of 5 seconds. I'm not sure if we could use the Stairway retry here or if it would be better to directly code it. In any case,